<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_sclimport</class>
 <widget class="QWidget" name="view_sclimport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>view_sclimport</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_open">
         <property name="text">
          <string>打开SCL文件(&amp;O)...</string>
         </property>
         <property name="icon">
          <iconset resource="sim_iec61850.qrc">
           <normaloff>:/sim_iec61850/Resources/OPENFOLD.ICO</normaloff>:/sim_iec61850/Resources/OPENFOLD.ICO</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sel_all">
         <property name="text">
          <string>全选(&amp;A)</string>
         </property>
         <property name="icon">
          <iconset resource="sim_iec61850.qrc">
           <normaloff>:/sim_iec61850/Resources/20070402165933688.gif</normaloff>:/sim_iec61850/Resources/20070402165933688.gif</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sel_none">
         <property name="text">
          <string>全不选(&amp;N)</string>
         </property>
         <property name="icon">
          <iconset resource="sim_iec61850.qrc">
           <normaloff>:/sim_iec61850/Resources/20070402165933865.gif</normaloff>:/sim_iec61850/Resources/20070402165933865.gif</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sel_rev">
         <property name="text">
          <string>反选(R)</string>
         </property>
         <property name="icon">
          <iconset resource="sim_iec61850.qrc">
           <normaloff>:/sim_iec61850/Resources/20070402165926456.gif</normaloff>:/sim_iec61850/Resources/20070402165926456.gif</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_import">
         <property name="text">
          <string>导入(&amp;I)</string>
         </property>
         <property name="icon">
          <iconset resource="sim_iec61850.qrc">
           <normaloff>:/sim_iec61850/Resources/export2.gif</normaloff>:/sim_iec61850/Resources/export2.gif</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sim_iec61850.qrc"/>
 </resources>
 <connections/>
</ui>
